elements/rh-accordion/docs/10-style.md

```diff
@@ -1,3 +1,5 @@
+<script type="module">import '/assets/elements/uxdot-code-sample.js';</script>
```

Comment:
Would it be possible to move this into the sample script?

Reply:
Fair point, I had thought about #1 but like you said the browser should in theory be able to handle that but it doesn't really impact anything other than needing to remember to put that line on a page when the code sample import is being used.